scripts/Avalanche.postman_environment.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
Since we're going to continue referencing `https://github.com/ava-labs/avalanche-postman-collection` you can delete this file.